File: Easy/Python/832. Flipping an Image.py
class Solution(object):
    def flipAndInvertImage(self, image):
        for col in image:
            i = 0
            j = len(image) - 1

            while i <= j:
                temp = col[i]
                col[i] = 1 - col[j]
                col[j] = 1 - temp
                i += 1
                j -= 1
        
        return image

File: Easy/Java/832. Flipping an Image.java
class Solution {
    public int[][] flipAndInvertImage(int[][] image) {
        for(int[] col : image){
            int i = 0;
            int j = image.length - 1;
            
            while(i <= j){
                int temp = col[i];
                col[i++] = 1 - col[j];
                col[j--] = 1 - temp;
            }
        }

        return image;
    }
}
